storage/retention.go

```diff
@@ -153,7 +153,7 @@ func (s *retentionEnforcer) expireData(ctx context.Context, buckets []*influxdb.
 		span.LogKV(
 			"bucket", b.ID,
 			"org_id", b.OrgID,
-			"system", b.IsSystem(),
+			"system", b.Type,
```

Comment:
nit: I'd recommend `system_type` for consistency with the log message. Would be worth updating `bucket` → `bucket_id` too.